test/powershell/engine/Help/UpdatableHelpSystem.Tests.ps1

```diff
@@ -0,0 +1,268 @@
+﻿
+# Test Settings:
+# This is the list of PowerShell Core modules for which we test update-help
+$moduleList = @(
+    "Microsoft.PowerShell.Archive"
+    "Microsoft.PowerShell.Host"
+    "Microsoft.PowerShell.Core"
+    "Microsoft.PowerShell.Diagnostics"
+    "Microsoft.PowerShell.Management"
+    "Microsoft.PowerShell.Security"
+    "Microsoft.PowerShell.Utility"
+    "PackageManagement"
+    "PowershellGet"
+    "PSReadline"
+)
+
+# The file extension for the help content on the Download Center.
+# For Linux we use zip, and on Windows we use $extension.
+$extension = ".zip"
+
+if ([System.Management.Automation.Platform]::IsWindows)
+ {
+    $moduleList += @(
+        "CimCmdlets"
+        "Microsoft.WsMan.Management"
+        "Microsoft.PowerShell.LocalAccounts"
+    )
+    $extension = ".cab"
+ }
+
+$expectedHelpFiles = @{}
```

Comment:
I am creating one hash table which contains all the expected information for each module. This looks like this:

$expectedFiles = @{

    CimCmdlets = @{
        HelpFiles = "Microsoft.Management.Infrastructure.CimCmdlets.dll-help.xml"
        HelpInfoFiles = "CimCmdlets_fb6cc51d-c096-4b38-b78d-0fed6277096a_HelpInfo.xml"
        CompressedFiles = "CimCmdlets_fb6cc51d-c096-4b38-b78d-0fed6277096a_en-US_HelpContent$extension"
    }
...